runtime/onert/backend/trix/ops/BulkLayer.h

```diff
@@ -50,7 +50,7 @@ class BulkLayer : public ::onert::exec::IFunction
   std::vector<const IPortableTensor *> _inputs;
   std::vector<IPortableTensor *> _outputs;
 
-  uint32_t _model_id;
+  std::vector<uint32_t> _model_id;
```

Comment:
Q. Do we need to make multiple `_model_id` for identical models? Is it now allowed to use same `_model_id` for same model (`.tvn`)?

Reply:
AFAIK, each core creates unique model_id for each request.

Reply:
No. It seems to make unique `req_id` for each request (via `createNPU_request`). `model_id` is for each model (via `registerNPUmodel`).

Reply:
@jyoungyun Do you have any information about https://github.com/Samsung/ONE/pull/9668#discussion_r963219378?